1039-wizard-design-guidance.md

```diff
@@ -0,0 +1,31 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-17
+updated: 2024-10-21
+---
+
+# Wizard Component
+
+A [Wizard](https://clarity.design/documentation/wizard) is a component that guides the user through a set of predefined steps. Wizards can help users focus on only the input needed for each step and decrease the chance of errors.
+
+## Guidance
+
+### Wizard Usage
+
+- **Must** use a wizard to help users walk through a defined step-by-step process.
+- **Must** use a wizard when the content is self-sufficient (no need to access other parts of the app) since it blocks access to other parts of the application that are necessary for completing the process.
```

Comment:
```suggestion
- **Should** use a wizard when the content is self-sufficient (no need to access other parts of the app) since it blocks access to other parts of the application that are necessary for completing the process.
```

I think this one should be a "should" because it's not required to use a wizard when the content is self-sufficient. A stepper might be self-sufficient. That doesn't seem like a reason to *require* it to be changed to a wizard.

Reply:
Makes sense